ibis/tests/backends.py

```diff
@@ -12,6 +13,8 @@
 from ibis.bigquery.tests.conftest import connect as bigquery_connect
 from ibis.impala.tests.conftest import IbisTestEnv as ImpalaEnv
 
+sa = pytest.importorskip('sqlalchemy')
```

Comment:
Can you remove this line and I will fix in another PR? I think we should skip in the `connect` method.

Reply:
Otherwise we end up skipping _every_ backend test instead of just the ones that need `sqlalchemy`.

Reply:
sounds good! I will remove it from here. thanks!